spec/spec_helper.rb

```diff
@@ -21,6 +21,7 @@
 
 require 'database_cleaner/active_record'
 require 'webmock/rspec'
+require 'rack_session_access/capybara'
```

Comment:
Remember to remove this along with the other pieces of the `rack_session_access` gem.